### ISSUE TITLE
gui/teleport doesn't work if the sidebar is hidden

### ISSUE THREAD

**Issue by alexchandel** [DFHack/dfhack#1186]
DFHack's `teleport` is indispensable right now for working around [the tree bug](http://www.bay12games.com/dwarves/mantisbt/view.php?id=9252) that's infected DF for the last two years. While the `teleport` function works, the GUI is broken:

```
[DFHack]# gui/teleport
./hack/lua/gui.lua:114: attempt to index a nil value (local 'rect')
stack traceback:
	./hack/lua/gui.lua:114: in function 'gui.inset_frame'
	./hack/lua/gui/dwarfmode.lua:389: in method 'computeFrame'
	./hack/lua/gui.lua:439: in field 'updateLayout'
	./hack/lua/gui/dwarfmode.lua:275: in method 'updateLayout'
	./hack/lua/gui.lua:566: in method 'onResize'
	./hack/lua/gui.lua:504: in local 'fun'
	./hack/lua/class.lua:98: in upvalue 'invoke_after_rec'
	./hack/lua/class.lua:94: in upvalue 'invoke_after_rec'
	./hack/lua/class.lua:94: in upvalue 'invoke_after_rec'
	./hack/lua/class.lua:94: in upvalue 'invoke_after_rec'
	./hack/lua/class.lua:129: in global 'TeleportSidebar'
	...f-fortress/0.43.05/libexec/hack/scripts/gui/teleport.lua:133: in local 'script_code'
	./hack/lua/dfhack.lua:562: in function 'dfhack.run_script_with_env'
	(...tail calls...)
```

**Comment by lethosor** [comment]
Does this happen right when you run that command or at some later stage? Are you using DFHack 0.43.05-r2?

**Comment by lethosor** [comment]
Oh, are you running this with the sidebar menu hidden?

**Comment by alexchandel** [comment]
As soon as I attempt to run the command, 0.43.05-r2, and yes I believe the sidebar was hidden. Does it have to be?

**Comment by lethosor** [comment]
It apparently does not work with the sidebar hidden. I thought it uses to, so I'll look into it.

**Comment by lethosor** [comment]
Alternatively, press "v" before running the script. That should show the sidebar temporarily.

**Comment by lethosor** [comment]
Fixed in #1251
